lib/BackgroundJob/ResetAssignedSignalingServer.php

```diff
@@ -43,6 +44,7 @@ public function __construct(ITimeFactory $time,
 
 		// Every 5 minutes
 		$this->setInterval(60 * 5);
+		$this->setTimeSensitivity(IJob::TIME_INSENSITIVE);
```

Comment:
```suggestion
		$this->setTimeSensitivity(IJob::TIME_SENSITIVE);
```
This needs to execute often